### PR TITLE
Make housing characteristics format consistent.

### PR DESCRIPTION
**Pull request by afontani**
## Pull Request Description

In the effort to make the format more consistent, the following format has been applied to the housing characteristics. Standardizing the format of the housing characteristics should make it easier to see differences in commits

- All line terminators are '\r\n' which is consistent with Windows machines. 
- All Option columns have format '%.6f'. 
- All [For Reference Only] Source Sample Size columns are type 'int'. 
- All [For Reference Only] Source Weight columns are type 'int'.

## Checklist

Not all may apply:

- [x] Unit tests have been added or updated
- [x] All rake tasks have been run, and pass
- [ ] PAT project measures/outputs have been updated
- [x] Documentation has been modified appropriately
- [ ] Any new options are added to `project_testing`
- [ ] `project_testing` runs without any failures
- [ ] No unexpected regression test changes
- [ ] All tests are passing (green) on circleci
- [x] The [changelog](https://github.com/NREL/OpenStudio-BuildStock/blob/master/CHANGELOG.md) has been updated appropriately
- [x] This branch is up-to-date with master

For more information on how to perform these checklist items, see the documentation's [Advanced Tutorial](https://resstock.readthedocs.io/en/latest/advanced_tutorial/index.html).

### CODE REVIEW

**Comment by shorowit** [comment]
Perhaps formatting should be enforced in the integrity_checks?

**Comment by afontani** [comment]
@shorowit Good idea. Thanks for the feedback.  I will update the integrity checks.

File: docs/read_the_docs/source/advanced_tutorial/writing_housing_characteristics.rst [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+This section provides a description the standard format for the housing characteristics. In order to stop recommitting entire files and to keep differences easy to read during the review process, a standard format for writing housing characteristics has been created. All housing characteristics shall follow the following format guidelines.
+
+Guidelines
+=============
+
+1. All line terminators are '\\r\\n'.
+2. All Option columns have format '%.6f'.
+3. All [For Reference Only] Source Sample Size columns are type 'int'.
+4. All [For Reference Only] Source Weight columns are type 'int'.
```
Comment (ejhw) [inline docs/read_the_docs/source/advanced_tutorial/writing_housing_characteristics.rst]: Add guidelines on "Created by" line

Comment (ejhw) [reply docs/read_the_docs/source/advanced_tutorial/writing_housing_characteristics.rst]: - Row values should sum to 1.000000.
- All combinations of dependencies are required to have values

Comment (shorowit) [reply docs/read_the_docs/source/advanced_tutorial/writing_housing_characteristics.rst]: Are you documenting what the integrity_checks do? There are additional checks (like [this one](https://github.com/NREL/OpenStudio-BuildStock/pull/313)).

Comment (afontani) [reply docs/read_the_docs/source/advanced_tutorial/writing_housing_characteristics.rst]: @ejhw @shorowit Once I have the integrity checks updated, I will make these updates.  Thank you for the review and providing an example.

**Comment by joseph-robertson** [comment]
I suggest we get this PR merged before merging any of: #356, #357, #358. This way we can isolate changes that #356, #357, #358 introduce.

@afontani Do you have more changes to make here?

**Comment by afontani** [comment]
@joseph-robertson I still need to make changes to the integrity checks that check for these changes.

**Comment by afontani** [comment]
@joseph-robertson Thanks for getting this cleaned up.

**Pull request merged**